templates/zerver/help/keyboard-shortcuts.md

```diff
@@ -8,103 +8,132 @@ Zulip keyboard shortcuts are divided into four categories:
 * [Navigation](#navigation)
 * [Composing messages](#composing-messages)
 * [Narrowing](#narrowing)
+* [Message actions](#message-actions)
 * [Menus](#menus)
+* [Drafts](#drafts)
 
 !!! warn ""
-    Note: We note how to enter keyboard shortcuts using keys not
-    present on MacOS keyboard in parentheticals below.
+    **Note:** We note how to enter keyboard shortcuts using keys not
+    present on Mac OS keyboard in parentheses below.
```

Comment:
'MacOS'/'macOS' is the correct one [1]. All the occurrences of `mac os`/`Mac OS X`/`Mac OS` in the code+doc should be replaced, except when a version is explicitly stated, then <10.11 should be kept at 'Mac OS X'.

[1] https://www.apple.com/macos/sierra/